packages/hooks/package.json

```diff
@@ -13,8 +13,19 @@
     "dist/*"
   ],
   "devDependencies": {
+    "@apollo/client": "^3.3.16",
```

Comment:
I think this should be added to `dependencies`. The same for `"graphql"`, and `"graphql-tag"`.

Reply:
@hosseintalebi It's already peer dependency, not an actual dependency of the library. Unless I'm misunderstanding peer dependencies.